smarts/core/mission_planner.py

```diff
@@ -423,6 +424,10 @@ def uturn_waypoints(self, sim, pose: Pose, vehicle):
                 speed_limit = neighborhood_vehicles[0].speed
 
         self._task_is_triggered = True
+        # uturn_activated_distance = math.sqrt((neighborhood_vehicles[0].pose.position[0]-ego_position[0])**2
+        #     +(neighborhood_vehicles[0].pose.position[1]-ego_position[1])**2
+        # )
+        # print(f"uturn activated at distance: {uturn_activated_distance}")
```

Comment:
Is this needed or just for debug?

Reply:
Actually both, it would be beneficial to log the distance between the u-turn vehicle and neighbor vehicle when u-turn is activated. I will use a logger for this. However

```
        uturn_activated_distance = math.sqrt((neighborhood_vehicles[0].pose.position[0]-ego_position[0])**2
            +(neighborhood_vehicles[0].pose.position[1]-ego_position[1])**2
        )
        print(f"uturn activated at distance: {uturn_activated_distance}")